.clang-format

```diff
@@ -1,101 +1,141 @@
 ---
```

Comment:
Can you also remove the seemingly random whitespace between keys and their values?